### ISSUE TITLE
Empty pubs repository: 0 division error with statistics

### ISSUE THREAD

**Issue by wschuell** [pubs/pubs#164]
If the repository is empty, pubs statistics raises a zero division error

```
➜  ~ pubs statistics      
error: float division by zero
Repository statistics:
➜  ~
```

**Comment by benureau** [comment]
Thanks William! That's obviously something we did not check for. Will be fixed very soon.

**Comment by benureau** [comment]
Ok it's fixed. The patch should be merged in master soon. Henceforth, you'll get:
```
$ pubs statistics 
Your pubs repository is empty.
```
I'll have to think about a witty remark to add to that.

**Comment by wschuell** [comment]
Or maybe add useful info like:
```
Your pubs repository is empty.
You can add or import new entries using 'pubs add' or 'pubs import'.
If you want to synchronize your repository between different computers, simply synchronize the '$HOME/.pubs' repository. But you will still need to execute 'pubs init' once on each computer.
```
The last sentence could be removed if during install (or at first run if not found) the .pubsrc is automatically created; but that would be another issue.

**Comment by benureau** [comment]
Hum. A link towards the first steps section of the documentation could do that. But let's be real: only very specific persons will use `pubs statistics` before `pubs add`. Congratulation William, you made the cut! :)

**Comment by benureau** [comment]
Oh, and you can place the `.pubs` directory anywhere you want, for instance in your <sync provider of you choice> folder. Or even move it after the fact, if you edit your `.pubsrc` appropriately.

**Comment by benureau** [comment]
Included in version 0.8.1. Closed by 8fe24140.
